functions/src/index.ts

```diff
@@ -15,6 +17,21 @@ initializeApp({
   credential: credential.cert(serviceAcc),
 });
 const database = firestore();
+const schema = Joi.object({
+  username: Joi.string(),
+  todayWork: Joi.string().required().messages({
+    "string.required": "Today's work field is required",
+    "string.min": "Today's work need's at least 1 character",
+  }),
+  yesterdayWork: Joi.string().required().messages({
+    "string.required": "Yesterday's work field is required",
+    "string.min": "Yesterday's work need's at least 1 character",
+  }),
+  date: Joi.date().min(new Date(2010, 1, 1)).required().messages({
+    "date.required": "Date field can't be empty",
+    "date.min": "Date field Should not be empty or older than 2010-01-01",
```

Comment:
Can we have a separate file for messages?

Reply:
Done